README.md

```diff
@@ -32,6 +32,10 @@ Additional rosdep sources.
 Don't add packages.ros.org as an apt repository.
 This allows to build against snapshots.ros.org, for example.
 
+## `SKIP_PACKAGES`
+
+Whitespace separated list of ROS package names not to be build.
```

Comment:
Could you add a note here about dependent packages failing to build if they depend on a package that's been skipped?

Something like "Note that if you skip a package, any of its dependent packages will fail to build, add any dependent package explicitly here to avoid that.